go.mod

```diff
@@ -5,22 +5,34 @@ go 1.14
 require (
 	github.com/Pallinder/go-randomdata v1.2.0
 	github.com/atomix/go-client v0.4.1
+	github.com/bshuster-repo/logrus-logstash-hook v1.0.0 // indirect
+	github.com/bugsnag/bugsnag-go v2.1.1+incompatible // indirect
+	github.com/bugsnag/panicwrap v1.3.2 // indirect
 	github.com/cenkalti/backoff v2.2.1+incompatible
 	github.com/docker/docker v1.13.1 // indirect
+	github.com/docker/go-metrics v0.0.1 // indirect
+	github.com/docker/libtrust v0.0.0-20160708172513-aabc10ec26b7 // indirect
+	github.com/garyburd/redigo v1.6.2 // indirect
+	github.com/gofrs/uuid v4.0.0+incompatible // indirect
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/mock v1.4.4
 	github.com/golang/protobuf v1.4.3
+	github.com/google/pprof v0.0.0-20190723021845-34ac40c74b70 // indirect
 	github.com/google/uuid v1.1.2
 	github.com/googleapis/gnostic v0.3.0 // indirect
+	github.com/gorilla/handlers v1.5.1 // indirect
+	github.com/gorilla/mux v1.8.0 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
+	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0 // indirect
+	github.com/kr/pty v1.1.8 // indirect
 	github.com/mattn/go-colorable v0.1.4 // indirect
 	github.com/mattn/go-isatty v0.0.12 // indirect
 	github.com/onosproject/config-models/modelplugin/devicesim-1.0.0 v0.6.28
 	github.com/onosproject/config-models/modelplugin/testdevice-1.0.0 v0.6.28
 	github.com/onosproject/config-models/modelplugin/testdevice-2.0.0 v0.6.28
 	github.com/onosproject/helmit v0.6.12
-	github.com/onosproject/onos-api/go v0.7.38
+	github.com/onosproject/onos-api/go v0.7.43
 	github.com/onosproject/onos-config-model v0.4.3
 	github.com/onosproject/onos-lib-go v0.7.10
```

Comment:
can you update this to `v0.7.12` too if you don't mind - same in `onos-config-model` too